src/chains/models.py

```diff
@@ -37,7 +37,11 @@ class RpcAuthentication(models.TextChoices):
         default=RpcAuthentication.NO_AUTHENTICATION,
     )
     safe_apps_rpc_uri = models.URLField(default="")
-    block_explorer_uri = models.URLField()
+    block_explorer_uri = (
+        models.URLField()
+    )  # TODO: Remove in a later migration (once the clients no longer need it)
```

Comment:
Couldn't we technically already remove this? Technically this would break the GW  but, we could open the PR there too and merge more or less at the same time to minimize down time. As this is only affecting staging anyhow.

Reply:
I agree. This request came from the clients though after staging broke last time (which I think it's reasonable) and since the field name was different I decided to remove it later.